-1

Tested at: 0275564811acb3d83ad3adbd4702f7dbd54db7c2

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22404/26412/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_1_X_2018-03-01-1100'
config/SCRAM/GMake/Makefile.rules:2097: recipe for target 'src' failed
gmake: *** [src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre>

--- a/EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py
+++ b/EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py
@@ -71,7 +71,7 @@ totemDAQMappingESSourceXML_TimingDiamond = cms.ESSource("TotemDAQMappingESSource
       validityRange = cms.EventRange("292521:min - 310000:max"),
       mappingFileNames = cms.vstring("CondFormats/CTPPSReadoutObjects/xml/mapping_timing_diamond_2017.xml"),
       maskFileNames = cms.vstring()
-    )
+    ),
     # 2018
     cms.PSet(
       validityRange = cms.EventRange("310001:min - 999999999:max"),

